### PR TITLE
VP-1395 Improve State tags on OpCard

### PR DESCRIPTION
**Pull request by waltzaround**
## I do solemnly swear that I have:
- [x] Run `npm test` and all the tests pass.
- [x] Run `npm run lint` and there are no warnings.
- [x] Not decreased the overall test coverage?
- [x] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. Improve tags for OpCards

## Screenshots 📷

image

### CODE REVIEW

File: components/Op/OpCard.js [hunk 1]
```diff
@@ -40,10 +40,10 @@ const OpCard = ({ op }) => {
   const interestIcon = ((interest) => {
     if (!interest) { return '' }
     switch (interest.status) {
-      case 'interested': return <StyledIcon type='like' theme='twoTone' twoToneColor='#6549AA' />
-      case 'invited': return <StyledIcon type='message' theme='twoTone' twoToneColor='#fb0' />
-      case 'committed': return <StyledIcon type='check-circle' theme='twoTone' twoToneColor='#0f0' />
-      case 'declined': return <StyledIcon type='close-circle' theme='twoTone' twoToneColor='#f00' />
+      case 'interested': return <TagState style={{ color: '#222', backgroundColor: '#E1E1E1' }}><StyledIcon type='mail' />You offered to help</TagState>
+      case 'invited': return <TagState style={{ color: 'white', backgroundColor: '#653CAD' }}><StyledIcon type='calendar' />You are invited</TagState>
+      case 'committed': return <TagState style={{ color: 'black', backgroundColor: '#36F482' }}><StyledIcon type='check-circle' />Accepted</TagState>
```
Comment (avowkind) [inline components/Op/OpCard.js]: At some point we can move these colour definitions to the theme.

File: components/Op/OpCard.js [hunk 2]
```diff
@@ -40,10 +40,10 @@ const OpCard = ({ op }) => {
...
+      case 'declined': return <TagState style={{ color: 'white', backgroundColor: '#F44336' }}><StyledIcon type='close-circle' />Cancelled</TagState>
```
Comment (avowkind) [inline components/Op/OpCard.js]: declined doesn't really mean that the op was cancelled. it may be going ahead but the person has been rejected.

Comment (waltzaround) [reply components/Op/OpCard.js]: Yeah, we should probably come up with some better language

File: components/Op/OpCard.js (hunk 1 above)
Comment (waltzaround) [reply components/Op/OpCard.js]: Can do - just need to move the palette from storybook into css variables

**Pull request merged**